src/main/java/de/tum/in/www1/artemis/web/rest/ModelingExerciseResource.java

```diff
@@ -292,7 +296,7 @@ public ModelingExerciseResource(ModelingExerciseRepository modelingExerciseRepos
 
     @GetMapping("/assessment-editor/{exerciseId}/{submissionId}")
     @PreAuthorize("hasAnyRole('TA', 'INSTRUCTOR', 'ADMIN')")
-    @Transactional(readOnly = true)
+    @Transactional
     @Timed
```

Comment:
Can you add JavaDoc comment what this method does? In particular if you now do a write transaction, it should be clear from the JavaDoc comments